<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MdiEvents</class>
 <widget class="QMainWindow" name="MdiEvents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Event"/>
   <addaction name="actionRemove_Event"/>
  </widget>
  <action name="actionNew_Event">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/png/icons/calendar--plus.png</normaloff>:/png/icons/calendar--plus.png</iconset>
   </property>
   <property name="text">
    <string>New Event</string>
   </property>
  </action>
  <action name="actionRemove_Event">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/png/icons/calendar--minus.png</normaloff>:/png/icons/calendar--minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove Event</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
